internal/profile/profile.go

```diff
@@ -12,6 +12,11 @@ import (
 	"github.com/ActiveState/cli/internal/errs"
 )
 
+var profilingEnabled = false
+
+func init() {
+	profilingEnabled = os.Getenv(constants.ProfileEnvVarName) == "true"
```

Comment:
nit: We do not need an `init()` function for this initialization.

Reply:
I could swear Go didn't let you call functions outside of a function.. not sure why this works.